src/nvim/ui.c

```diff
@@ -166,6 +167,88 @@ void ui_event(char *name, Array args)
   }
 }
 
+/// @param[in] aep data to convert
+/// @param[out] out structure that will be sent to UIs
+HlAttrs attrentry2hlattrs(const attrentry_T *aep, bool use_rgb)
```

Comment:
param doc is wrong

Reply:
this is borrowed code from @teto:s PR that will disappear on rebase.